src/inmanta/agent/config.py

```diff
@@ -160,6 +160,16 @@
     is_float,
 )
 
+agent_virtual_environment_cleanup: Option[int] = Option(
+    "config",
+    "agent-venv-cleanup",
+    7,
+    "This is the number of days to wait before unused Python virtual environments are cleaned from the inmanta "
+    "server. Setting this option too low may result in a high load on the Inmanta server. Setting it too high"
+    " may result in long deployment times.",
+    is_int,
```

Comment:
I wonder if we can make it float? (if you want to go really short, it is easy for us to allow and may come in handy)